### PR TITLE
Admin -> Pages -> Edit Path -> Save/Update

### PR DESCRIPTION
**Pull request by CubE135**
**Reasoning for Pull Request:**
As i was confused by the edit icons for the path and locale when editing a page, i decided to add some function to it.

**Changes:**
For now, i added a "coming soon" notification to the locale edit icon and focused on the path editing only. But i prepared a bit for adding locale editing in the near future.
When clicking the path edit icon, the user will be presented with a input field. The user can then edit the path to his liking.
After editing the path, the user will need to save the changes. I added a "Update Page" Button to the top right of the page.
Pressing that button will save the changed path to the database.
For editing other values on that page, it is now very easy to add more editable values in the future.

**Code clarification:**
For some reason, when trying to update the page through a mutation, i was told that the page content can not be empty. That doesnt really make sense when editing only the path, so i made it possible to edit the page without providing page-content.

I also ran into an error, when editing the page, because of the tags.
Sorrounding the `associateTags-call` with a simple "if" solved the issue.

**Locales:**
As i had to add some new texts, these are the language keys i added:
```
common:
  page.updatePage: 'Update Page'
  page.pageUpdateSuccess: 'Page updated successfully.'
```

Best Regards,
CubE :)

### CODE REVIEW

File: client/components/admin/admin-pages-edit.vue [hunk 1]
```diff
@@ -165,7 +187,9 @@
 <script>
 import _ from 'lodash'
 import { StatusIndicator } from 'vue-status-indicator'
+import gql from 'graphql-tag'
 
+import localesQuery from 'gql/admin/locale/locale-query-list.gql'
```
Comment (NGPixel) [inline client/components/admin/admin-pages-edit.vue]: You actually don't need to remotely fetch the installed locales, you have them already in the `siteLangs` variable, which is available everywhere.

Example value:
```
[
  {
    "code":"en",
    "name":"English"
  },
  {
    "code":"fr",
    "name":"français"
  }
]
```

File: client/components/admin/admin-pages-edit.vue [hunk 2]
```diff
@@ -212,6 +247,77 @@ export default {
...
+        _.delay(() => {
+          this.$refs[ipt].focus()
+        }, 200)
+      })
+    },
+    /**
+     * Update the current page
+     */
+    async updatePage() {
+      this.$store.commit(`loadingStart`, 'admin-users-update')
```
Comment (NGPixel) [inline client/components/admin/admin-pages-edit.vue]: Make sure all the loading triggers have a proper ID.

**Comment by NGPixel** [comment]
Maybe calling the pages > move mutation would make more sense, rather than update:
https://github.com/Requarks/wiki/blob/master/server/graph/schemas/page.graphql#L95-L99

File: client/components/admin/admin-pages-edit.vue (hunk 1 above)
Comment (CubE135) [reply client/components/admin/admin-pages-edit.vue]: Oh ok, i will fix that :)

File: client/components/admin/admin-pages-edit.vue (hunk 2 above)
Comment (CubE135) [reply client/components/admin/admin-pages-edit.vue]: True, will be fixed :)

**Comment by CubE135** [comment]
> Maybe calling the pages > move mutation would make more sense, rather than update:
> https://github.com/Requarks/wiki/blob/master/server/graph/schemas/page.graphql#L95-L99
I thoght, the feature may be extended at some point in time, where other values on that page would be editable too. Then, we would need the update mutation anyways, right?

**Comment by NGPixel** [comment]
Good point.

**Comment by CubE135** [comment]
Fixed the things you mentioned :)

**Comment by CubE135** [comment]
Hey :) Is there anything else i need to fix?

**Pull request closed without merging**